.storybook/addons/google-analytics/register.js

```diff
@@ -0,0 +1,32 @@
+import { addons } from '@storybook/manager-api';
+import { STORY_CHANGED, STORY_ERRORED, STORY_MISSING } from '@storybook/core-events';
+import ReactGA from 'react-ga4';
+
+addons.register('storybook/google-analytics', (api) => {
+  // Do not do anything if running in localhost
+  if (globalThis.location.hostname === 'localhost') {
```

Comment:
We may want to improve this check, but for now it should work fine to avoid GA events being sent from dev environments.